src/main/kotlin/me/lensvol/blackconnect/PluginStartupActivity.kt

```diff
@@ -19,5 +30,36 @@ class PluginStartupActivity : StartupActivity, DumbAware {
         } else {
             logger.info("Unknown version of BlackConnect plugin is ready to start.")
         }
+
+        if (globalSettings.spawnBlackdOnStartup) {
```

Comment:
It starts blackd when running any unit tests. I would skip this part in unittests since we assume there that we start blackd manually (at least for now). See `ApplicationManager.getApplication().isUnitTestMode`.

By the way, should be enable tests in GitHub actions for pull requests?

Reply:
Yes, I think it is a good idea to also run them in PRs.